ctdl/ctdl.py

```diff
@@ -209,6 +212,9 @@ def main():
 	parser.add_argument("-nr", "--no-redirects", action = 'store_true', default = False,
 						help = "Prevent download redirects.")
 
+	parser.add_argument("-w", "--website", default = "google.com", type = str,
```

Comment:
Setting `default` as `google.com` returns results from google websites only.
Make it `None`. In case of `None`, dont't use site argument in google search query.